charts/linkerd-crds/templates/gateway.networking.k8s.io_grpcroutes.yaml

```diff
@@ -0,0 +1,2258 @@
+{{ if (and .Values.enableGrpcRoutes (include "partials.crdInstalled" "grpcroutes.gateway.networking.k8s.io" | empty)) }}
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
lets copy this verbatim without any formatting changes from https://github.com/kubernetes-sigs/gateway-api/blob/main/config/crd/experimental/gateway.networking.k8s.io_grpcroutes.yaml (with the exception that we need to add the Linkerd labels and annotations like you've already done here)